### DISCUSSION TITLE
关于 Umi-OCR  卸载/删除/更新

### DISCUSSION THREAD

**Discussion by xinzhoushiyuan** [hiroi-sora/Umi-OCR#458] [Q&A]
如果想要安装新版本的Umi-OCR 只需要在资源管理器将Umi-OCR 解压出来的那个文件夹Umi-OCR_Rapid_v2.1.0.7z 删除就行了吗

**Comment by hiroi-sora** [comment]
是滴，Umi-OCR是绿色（便携）软件，不需要安装/卸载，只需要解压/删除文件。
